modules/ossm-config-otel.adoc

```diff
@@ -99,41 +99,51 @@ spec:
 Once you verify that you can see traces, lower the `randomSamplingPercentage` value or set it to `default` to reduce the number of requests.
 ====
 
-. Deploy the `bookinfo` application in `bookinfo` namespace:
+. Create the `bookinfo` namespace by running the following command:
 +
 [source, terminal]
 ----
 $ oc create ns bookinfo
 ----
+
+. Depending on the update strategy you're using, enable sidecar injection in the namespace by running the appropriate command(s):
+
+.. If you are using the `InPlace` update strategy, run the following command:
```

Comment:
Maybe also note that this only works "if you are using the `InPlace` update strategy and your `Istio` resource has the name `default`."